docs/README.rst

```diff
@@ -65,6 +65,10 @@ Device configuration
 
 -  Take a video ``adbe screenrecord video.mp4 # Press ^C when finished``
 
+-  Switch Debug to Wireless mode
+
+``adbe wireless``
```

Comment:
Thanks. I just tried and it works seamlessly with a single device connected. Can you think of a better sub-command name? For example, `adbe enable wireless debugging`.

In the future, we can `adbe disable wireless debugging` as well.

Reply:
It's too long, you don't think? I use shell script with same functionality with short alias `adbw`, so it's very fast and simple to move device to wireless debug mode

Reply:
Command added

Reply:
You are right that this is long and I am open to alternatives but `adbe wireless` is too short. It can even mean that you are turning the wireless (Wi-Fi?) on/off of the device. And not the ADB debugging connection.

Reply:
First of all, user always can set his own aliases in bash/cmd environment. So we can set more informative commands and add shortcuts later

Reply:
Perfect. We are on the same page.